### DISCUSSION TITLE
Add a new category called 💻 Codes

### DISCUSSION THREAD

**Discussion by rishanbenahalkar** [SudhanPlayz/Discord-MusicBot#950] [Ideas]
This new category will help people who want to add a few handy commands to their bot while others post code, I personally have a lot of extra commands that I made for my own bot, and for some of them I really feel that they could help someone out or cheer them up with a few games. For reference, here are all the commands I made, `8ball.js | banner.js | eval.js | execute.js | feedback.js | configinfo.js | instagram.js | maintenance.js | removemaintenance.js | minesweeper.js | bugreport.js | weather.js | wordle.js`
if you find at least a few of them helpful consider this idea as something to implement!
I'm not just suggesting this because I want to flaunt the amount of commands I've got, but to help a few people who'd be happy to have these in their bot.

**Comment by DevAmiyo** [comment]
Well.. there's `🙌 Show and tell`. I think that has the same purpose.
